winterm/ansi.go

```diff
@@ -163,15 +163,28 @@ func ensureInRange(n int16, min int16, max int16) int16 {
 
 func GetStdFile(nFile int) (*os.File, uintptr) {
 	var file *os.File
-	switch nFile {
-	case syscall.STD_INPUT_HANDLE, windows.STD_INPUT_HANDLE:
+
+	// syscall uses negative numbers
+	// windows package uses very big uint32
+	// Keep these switches split so we don't have to convert ints too much.
+	switch uint32(nFile) {
```

Comment:
I went over a few variations as well; would casting to `int64` work? https://github.com/docker/cli/pull/3131/commits/97da45d9a187ae1d697ed2023bd05b0fd6c47602

Reply:
It does, but I figured it would be better to split since these are really different types.

Reply:
Yeah, I think ultimately we should drop the syscall one (and introduce a breaking change once)